src/index.ts

```diff
@@ -33,7 +33,18 @@ const isDevelopmentVersion = (version: string) => {
 	return /(\.|\+|-)dev/.test(version);
 };
 
-export const compare = memoize((versionA: string, versionB: string) => {
+export const compare = memoize((versionA: string | null, versionB: string | null) => {
+	if (versionA === null && versionB === null) {
```

Comment:
How about generalizing this as `if (versionA === versionB)` ?
But, maybe the explicit null checks keep the code's purpose clearer.